code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -27,6 +28,12 @@
 		stop_wagging_tail(H)
 	. = ..()
 
+/datum/species/human/felinid/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H, delta_time, times_fired)
```

Comment:
You're trying to negate damage done by carpotoxin by making it heal felinids the amount of damage done, but doing this will rely on either:
1) carpotoxin never being changed
2) whoever changes carpotoxin to know that felinids also have to be changed to match this

You should instead revert to what you did before, enriched electricity does the same thing for ethereals.

Reply:
True, this is a little too hacky. Gonna revert it to an on_mob_add for carpotoxin.